.obs/dockerfile/slem-base-iso/Dockerfile

```diff
@@ -25,16 +27,18 @@ COPY --from=builder /output /elemental-iso
 ARG SLEMICRO_VERSION
 ARG BUILD_REPO=%%IMG_REPO%%
 ARG IMAGE_REPO=$BUILD_REPO/suse/sle-micro-iso/base-$SLEMICRO_VERSION
+ARG IMAGE_TAG=%VERSION%-%RELEASE%
 
 # Define labels according to https://en.opensuse.org/Building_derived_containers
 # labelprefix=com.rancher.slem
 LABEL org.opencontainers.image.title="SLE Micro base-ISO"
-LABEL org.opencontainers.image.description="Includes the SLE Micro ISO"
-LABEL org.opencontainers.image.version="%VERSION%"
+LABEL org.opencontainers.image.description="Includes the SLE Micro base-ISO"
+LABEL org.opencontainers.image.version="${IMAGE_TAG}"
```

Comment:
Setting the version to the most specific image tag -> so even on re-tags we can easily relate it with some specific build.